tests/devapps/XamarinDev/XamarinDev/App.xaml.cs

```diff
@@ -13,18 +13,18 @@ public partial class App : Application
 
         public static object RootViewController { get; set; }
 
-        public const string DefaultClientId = "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0";
+        public const string DefaultClientId = "16dab2ba-145d-4b1b-8569-bf4b9aed4dc8";
         // For system browser
         //public const string DefaultClientId = "5a434691-ccb2-4fd1-b97b-b64bcfbc03fc";
 
         public const string B2cClientId = "e3b9ad76-9763-4827-b088-80c7a7888f79";
 
 
-        public static string RedirectUriOnAndroid = Microsoft.Identity.Client.Core.Constants.DefaultRedirectUri; // will not work with system browser
+        //public static string RedirectUriOnAndroid = Microsoft.Identity.Client.Core.Constants.DefaultRedirectUri; // will not work with system browser
         // For system browser
-        //public static string RedirectUriOnAndroid = "msauth-5a434691-ccb2-4fd1-b97b-b64bcfbc03fc://com.microsoft.identity.client.sample";
+        public static string RedirectUriOnAndroid = "msauth://com.microsoft.identity.client/2RJuUvgnnEn0C0I0IOd1qHqOUMg%3D";
 
-        public const string BrokerRedirectUriOnIos = "msauth.com.companyname.XamarinDev://auth";
+        public const string BrokerRedirectUriOnIos = "msauth://com.microsoft.identity.client/2RJuUvgnnEn0C0I0IOd1qHqOUMg%3D";
```

Comment:
>BrokerRedirectUriOnIos [](start = 28, length = 22)

does this still work w/ios broker?

Reply:
This was a mistake. it is my test redirect URI, will be reverted